crates/sui-core/src/verify_indexes.rs

```diff
@@ -88,3 +91,59 @@ pub fn verify_indexes(store: &dyn AccumulatorStore, indexes: Arc<IndexStore>) ->
 
     Ok(())
 }
+
+pub async fn fix_indexes(authority_state: Weak<AuthorityState>) -> Result<()> {
```

Comment:
Lets add a comment stating that this should be removed in release X or something to that effect so that when others see this they know the reasoning/rationale and that it is ok to nuke after a point in time.